source/JsMaterialX/test/codeExamples.spec.js

```diff
@@ -51,28 +51,28 @@ describe('Code Examples', () => {
         expect(inputValue).to.not.be.null;
         expect(inputValue.getData()).to.equal(0.25);
 
-        // Create a material that instantiates the shader.
-        const material = doc.addMaterial();
-        const materials = doc.getMaterials();
-        expect(materials.length).to.equal(1);
-        expect(materials[0]).to.eql(material);
-        const refSimpleSrf = material.addShaderRef('SR_simpleSrf', 'simpleSrf');
-        const shaderRefs = material.getShaderRefs();
-        expect(shaderRefs.length).to.equal(1);
-        expect(shaderRefs[0]).to.eql(refSimpleSrf);
-        expect(shaderRefs[0].getName()).to.equal('SR_simpleSrf');
-
-        // Bind roughness to a new value within this material.
-        const bindInput = refSimpleSrf.addBindInput('roughness');
-        const bindInputs = refSimpleSrf.getBindInputs();
-        expect(bindInputs.length).to.equal(1);
-        expect(bindInputs[0]).to.eql(bindInput);
-        bindInput.setValuefloat(0.5);
-        expect(bindInput.getValue()).to.not.be.null;
-        expect(bindInput.getValue().getData()).to.equal(0.5);
-
-        // Validate the value of roughness in the context of this material.
-        expect(roughness.getBoundValue(material).getValueString()).to.equal('0.5');
+        // // Create a material that instantiates the shader.
+        // const material = doc.addMaterial();
+        // const materials = doc.getMaterials();
+        // expect(materials.length).to.equal(1);
+        // expect(materials[0]).to.eql(material);
+        // const refSimpleSrf = material.addShaderRef('SR_simpleSrf', 'simpleSrf');
+        // const shaderRefs = material.getShaderRefs();
+        // expect(shaderRefs.length).to.equal(1);
+        // expect(shaderRefs[0]).to.eql(refSimpleSrf);
+        // expect(shaderRefs[0].getName()).to.equal('SR_simpleSrf');
+
+        // // Bind roughness to a new value within this material.
+        // const bindInput = refSimpleSrf.addBindInput('roughness');
+        // const bindInputs = refSimpleSrf.getBindInputs();
+        // expect(bindInputs.length).to.equal(1);
+        // expect(bindInputs[0]).to.eql(bindInput);
+        // bindInput.setValuefloat(0.5);
+        // expect(bindInput.getValue()).to.not.be.null;
+        // expect(bindInput.getValue().getData()).to.equal(0.5);
+
+        // // Validate the value of roughness in the context of this material.
+        // expect(roughness.getBoundValue(material).getValueString()).to.equal('0.5');
```

Comment:
Should this be removed instead of commented out, if the material element is now gone for good?

Reply:
I was thinking of removing, but it's more of remapping this test which can't be done until there are new Javascript bindings for materials.